### PR TITLE
instance pages

### PR DESCRIPTION
**Pull request by allenwq**
depends on #76 

Changs in this PR:

1. instance mangement actions new/create/edit/update/index

2. feature specs

### CODE REVIEW

**Comment by lowjoel** [comment]
Wait for my part to merge first, I still can't figure out all the different conventions.

**Comment by lowjoel** [comment]
Blocked by #66.

**Comment by wangqiang1208** [comment]
I think we need to discuss how to manage our views here.. instance management is the feature for super admin, we might not  want to mix admin and super admin pages/functions ?

**Comment by lowjoel** [comment]
I think it's fine. Only global admins have access to it, there's no reason for it to be separate. It appears on every instance admin's page if they are also a global admin. What do you guys thingk @jsyeo @weiqingtoh

**Comment by wangqiang1208** [comment]
Previously seems that I'm doing the same thing. But remember what prof Ben said on last meeting ? A person maybe the admin of serval instances, he wants us to provide a sliding down menu for each of these instances.

And how can an super admin access another instance's settings without going to that instance ?

**Comment by lowjoel** [comment]
I think having the drop-down only serves to confuse. If you're white labelling then you would identify the instance by the design of the layout.

If are admins of multiple instances you'll probably end up adminning the wrong instance if you determine the instance you are administrating by a dropdown.

Opinions?

**Comment by wangqiang1208** [comment]
So you think that admin need to go that instance to do administration ?

**Comment by lowjoel** [comment]
I think that reduces the problem of people accidentally doing admin things on the wrong instance... which is bad.

**Comment by wangqiang1208** [comment]
But on the header, there should be a instance name, this is what prof Ben
says on last meeting..  I think we may need to further discuss this with
him.

**Comment by lowjoel** [comment]
Isn't the point of white labelling not to have just a simple instance name...? The look should be substantially different.

Anyway, I think the changes between instances should be as obvious as possible, otherwise why are we spending all the effort and code to make the app a multitenant...

**Comment by lowjoel** [comment]
@wangqiang1208 I think we can revive this PR.

**Comment by wangqiang1208** [comment]
OK, I'll update the code here

File: app/views/admin/instances/_instance.html.slim [hunk 1]
```diff
@@ -0,0 +1,10 @@
+tr
+  td
+    a
+      = link_to instance.name, admin_instance_path(instance)
+
+  td= instance.host
```
Comment (lowjoel) [inline app/views/admin/instances/_instance.html.slim]: space after =

**Pull request merged**